src/settings/MatchProfiles/MatchProfiles.js

```diff
@@ -98,21 +107,15 @@ export const matchProfilesShape = {
       path: 'data-import-profiles/matchProfiles',
       clientGeneratePk: false,
       throwErrors: true,
-      GET: {
-        params: {
-          query: makeQueryFunction(
-            'cql.allRecords=1',
-            '(name="%{query.query}*" OR tags.tagList="%{query.query}*")',
-            {
-              name: 'name',
-              tags: 'tags.tagList',
-              updated: 'metadata.updatedDate',
-              updatedBy: 'userInfo.firstName userInfo.lastName userInfo.userName',
-            },
-            [],
-          ),
-        },
-        staticFallback: { params: {} },
+      params: (_q, _p, _r, _l, props) => {
+        const findAll = 'cql.allRecords=1';
+        const sort = get(_r, ['query', 'sort'], null);
+        const search = get(_r, ['query', 'query'], null);
+        const sortQuery = `${sort ? `sortBy ${getSortQuery(sortMap, sort)}` : ''}`;
```

Comment:
bit hard coded, it might be easier
```suggestion
        const sortQuery = sort ? `sortBy ${getSortQuery(sortMap, sort)}` : '';
```

Reply:
Forgot to change it, thanks!